crd/apis/network/v1/network_types.go

```diff
@@ -132,7 +132,13 @@ type NetworkSpec struct {
 	// IPAMMode specifies the IPAM mode for the network.
 	// Valid options include: internal, external
 	// +optional
+	// +deprecatedversion:warning="IPAMMode is deprecated in favor of IPAM4."
```

Comment:
is there a link to the rationale for this change, and how the transition from this field to the new field is planned?